src/v/serde/parquet/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ v_cc_library(
   NAME serde_parquet
   SRCS
     metadata.cc
+    encoding.cc
+    flattened_schema.cc
```

Comment:
>     flattened_schema.cc

introduced previous commit?

Reply:
Yeah the first PR I missed a source file when I had extracted out some functions out of headers